devtools/gradle/src/main/java/io/quarkus/gradle/tasks/QuarkusPlatformTask.java

```diff
@@ -107,70 +108,75 @@ private QuarkusPlatformDescriptor resolvePlatformDescriptor(QuarkusJsonPlatformD
         final Configuration boms = project.getConfigurations()
                 .detachedConfiguration(bomDeps.toArray(new org.gradle.api.artifacts.Dependency[0]));
         final Set<AppArtifactKey> processedKeys = new HashSet<>(1);
-        final List<ResolvedArtifact> descriptorDeps = new ArrayList<>(2);
+
+        final List<Dependency> descriptorDeps = new ArrayList<>();
+        final AtomicInteger quarkusBomIndex = new AtomicInteger(-1);
```

Comment:
Just curious: Why an `AtomicInteger` here instead of an `int` as used in the `QuarkusProjectMojoBase`?

Reply:
Its value is set in an anonymous class.